oleggerasimovich/src/parser.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import json
+import os
+from argparse import Namespace
+
+from dotenv import load_dotenv
+from pymongo import MongoClient
+
+def cli() -> Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('user_csv', type=str, help='Put path to ur csv file')
+    cli_arguments  = parser.parse_args()
+
+    return cli_arguments 
+
+
+def db_loader(path: str) -> None:
```

Comment:
Meanwhile, `db_loader` is not a very good name for the function. It is better to name functions using imperative mood like `load_to_mongo`.

 `DBLoader` is a good name for the object itself (e.g. class, or variable).